pep-0376.txt

```diff
@@ -1,15 +1,24 @@
 PEP: 376
 Title: Database of Installed Python Distributions
-Version: $Revision$
-Last-Modified: $Date$
 Author: Tarek Ziadé <tarek@ziade.org>
 Status: Final
 Type: Standards Track
 Topic: Packaging
 Content-Type: text/x-rst
 Created: 22-Feb-2009
 Python-Version: 2.7, 3.2
-Post-History:
+Post-History: `22-Jun-2009 <https://mail.python.org/archives/list/python-dev@python.org/thread/ILLTIOZAULMDY5CAS6GOITEYJ4HNFATQ/#ILLTIOZAULMDY5CAS6GOITEYJ4HNFATQ>`__
+
+
+.. important::
+   This PEP is a historical document; the up-to-date, canonical version
+   of the specification originally defined here is now
+   `maintained on the PyPA Specifications section of packaging.python.org
+   <https://packaging.python.org/en/latest/specifications/recording-installed-packages/>`__.
+   See the `PyPA specification update process
```

Comment:
Let’s add a link to the specification directly?

Reply:
I'm a little confused—is that not a direct link to the specification?

Or would you suggest I update the link text to be more clear as to its target, which I imagine might be the source of the confusion here?

Reply:
Bah, I didn't look closely enough.

Let's update the link text, yea.

Reply:
I forgot to note, sorry, but in the PyPA spec directive in #2702 which I'll update this PR to use when that goes in, I've addressed the originally unclear text by making the link title much more explicit that it is pointing to the specification, and also allowing authors to specify the name of the spec as the link text. I welcome further feedback there if you have additional suggestions on the text!

Reply:
I revised the wording in #2702 based on your feedback here and rebased this PR on that now that its merged, so let me know if this concern is resolved, or you have further feedback. Thanks!